ibis/backends/tests/test_client.py

```diff
@@ -385,31 +385,31 @@ def test_unsigned_integer_type(alchemy_con, coltype):
 @pytest.mark.parametrize(
     ("expr", "expected"),
     [
-        param(
-            ibis.table(data=[(1, 2.0, "3")], columns=list("abc")),
-            pd.DataFrame([(np.int8(1), 2.0, "3")], columns=list("abc")),
-            id="simple",
-        ),
-        param(
-            ibis.table(data=[(1, 2.0, "3")]),
-            pd.DataFrame(
-                [(np.int8(1), 2.0, "3")], columns=["col0", "col1", "col2"]
-            ),
-            id="simple_auto_named",
-        ),
-        param(
-            ibis.table(
-                data=[(1, 2.0, "3")],
-                schema=ibis.schema(dict(a="int8", b="float32", c="string")),
-            ),
-            pd.DataFrame([(1, 2.0, "3")], columns=list("abc")).astype(
-                {"a": "int8", "b": "float32"}
-            ),
-            id="simple_schema",
-        ),
+        # param(
+        #     ibis.table(data=[(1, 2.0, "3")], columns=list("abc")),
```

Comment:
These cases could be replaced by first constructing the dataframe from the plain python list.

Reply:
I think we should support this case and not make people call another function if we don't have to.

Reply:
How about overloading a new `ibis.memtable()` function to support these cases, but defer the data construction and datatype inference to pandas?

Reply:
Pandas can't infer complex types last I checked, so I don't think we can use it here.

Reply:
pyarrow should be able to do that though